db/migrate/20160626215946_create_keywords.rb

```diff
@@ -0,0 +1,7 @@
+class CreateKeywords < ActiveRecord::Migration
+  def change
+    create_table :keywords do |t|
```

Comment:
If these aren't surfaced in the UI and are just to improve search quality, I'd still hold off on introducing keywords till after a basic search feature is implemented and there's a clear need.